### PR TITLE
feat(sdk-coin-avaxp): fix verify transaction method

### PR DESCRIPTION
**Pull request by jrafael-simtlix**
Avaxp staking method does not send recipients

STLX-18437

### CODE REVIEW

**Review by jicli — DISMISSED** [review]
LGTM

File: modules/sdk-coin-avaxp/src/avaxp.ts [hunk 1]
```diff
@@ -52,31 +57,35 @@ export class AvaxP extends BaseCoin {
...
-        _.pick(output, ['address', 'amount'])
-      );
+    const { type, stakingOptions, memo } = params.txParams;
+    if (!type || explainedTx.type !== TransactionType[type]) {
+      throw new Error('Tx type does not match with expected txParams type');
+    }
+    if (memo && explainedTx.memo !== memo.value) {
+      throw new Error('Tx memo does not match with expected txParams memo');
+    }
+    if (!params.txParams.recipients || params.txParams.recipients.length === 0) {
```
Comment (CapnMigraine) [inline modules/sdk-coin-avaxp/src/avaxp.ts:81]: nit: nodeID is not really a recipient. 
Can we verify based on the tx type? We should only check `stakingOptions` when the tx is for a staking operation

**Review by CapnMigraine — APPROVED** [review]
lgtm

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Comment by mmcshinsky-bitgo** [comment]
@jrafael-simtlix looks like a merge conflict to resolve

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Pull request merged**